galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.gradle.manager/bnd.bnd

```diff
@@ -0,0 +1,6 @@
+-snapshot: ${tstamp}
```

Comment:
best placed in the languages parent,  not the testingtools parent

Reply:
Will fix.